docs/streams/upgrade-guide.html

```diff
@@ -52,6 +52,20 @@ <h1>Upgrade Guide and API Changes</h1>
         <li> restart all new ({{fullDotVersion}}) application instances </li>
     </ul>
 
+    <p>
+        As of Kafka Streams 2.6.x a new processing mode <code>"exactly_once_beta"</code> (configurable via parameter
+        <code>processing.guarantee</code>) is available.
+        To use this new feature, your brokers must be on version 2.5.x or newer.
```

Comment:
We could also mention the potential outcome if broker is not on >= 2.5.x, which is a hard crash in this transition.

Reply:
I guess we can. However, isn't a hard crash _always_ the result?